src/fees/AaveFlashLoanFeeCalculator.sol

```diff
@@ -20,31 +20,17 @@ contract AaveFlashLoanFeeCalculator is IFeeCalculator, FeeCalculatorBase {
 
     function getFees(address to, bytes calldata data) external view returns (IParam.Fee[] memory) {
         // Aave flash loan signature:'flashLoan(address,address[],uint256[],uint256[],address,bytes,uint16)', selector: 0xab9c4b5d
-        (, address[] memory tokens, uint256[] memory amounts, , , bytes memory params, ) = abi.decode(
+        (, address[] memory tokens, uint256[] memory amounts, , , , ) = abi.decode(
             data[4:],
             (address, address[], uint256[], uint256[], address, bytes, uint16)
         );
 
         amounts = calculateFee(amounts);
```

Comment:
Could remove if performing the calculation directly within the `_createFee()` function.